scala3doc/src/dotty/dokka/tasty/TypesSupport.scala

```diff
@@ -215,9 +215,12 @@ trait TypesSupport:
         //     case _ =>
         //     throw Exception("Match error in TypeRef. This should not happen, please open an issue. " + convertTypeOrBoundsToReference(reflect)(qual))
         // }
-      case tr @ TermRef(qual, typeName) => qual match {
-        case _ => link(tr.termSymbol)
-      }
+      case tr @ TermRef(qual, typeName) => 
+        tr.termSymbol.tree match
```

Comment:
This one here is a little bit tricky. The reason behind this is to extract `ConstantType` when one occurs